conduct.md

```diff
@@ -132,7 +132,13 @@ TODO: Write an introductory paragraph.
 
 ### Infractions
 
-TODO: Write an introductory paragraph.
+Infractions are violations of the class conduct guide as specified by the unacceptable conduct sections. It is important that the conduct guide is upheld and acceptable conduct is followed in order to support a positive learning environment in the class and online.
+
+When a violation is committed, the student is given a penalty for the total amount of violations committed during the semester.  The only exception to this is that Honor Code violations, and the penalties associated with them, are handled and investigated by the Honor Committee.
+
+If you believe that a violation was reported incorrectly or unfairly, it is possible to meet with the professor teaching the class to discuss a possible appeal for the violation.  After discussion of the violation, the professor will make the final decision about whether an appeal will be given.
```

Comment:
"whether an appeal will be given." is a bit confusing. Could this just be "the violation"? So "the professor will make the final decision about the violation".